src/components/coffee-shelf.json

```diff
@@ -0,0 +1,25 @@
+{
+  "Shelf": [
+    {
+      "coffee-name": "La Laguna", 
+      "roaster-name": "Kuma Coffee", 
+      "roast-date": 1495324800000, 
+      "coffee-size": "12 oz",
+      "gone": false
+    }, 
+    {
+      "coffee-name": "Timepiece", 
+      "roaster-name": "Tweed Coffee", 
+      "roast-date": 1494028800000, 
+      "coffee-size": "16 oz",
+      "gone": true
+    }, 
+    {
+      "coffee-name": "Lotus", 
+      "roaster-name": "Barrett's", 
+      "roast-date": 1492214400000,
+      "coffee-size": "16 oz",
+      "gone": true
+    }
+  ]
+}
```

Comment:
i think you can delete this file (or you can copy the data from the other shelf into it). if you copy that other data, then in `app-body.js` you would have

```diff
 import AddCoffeeForm from './temporary-add-to-db-component';
 import CoffeeShelf from './test-shelf.js';
+import testShelf from './test-shelf.json';
- 
-var testShelf = {
-  Shelf: [
 ...
```